spack/packages/dla-future/package.py

```diff
@@ -16,23 +15,40 @@ class DlaFuture(CMakePackage):
 
     version('develop', branch='master')
 
-    variant('gpu', default=False,
+    variant('cuda', default=False,
             description='Use the GPU/cuBLAS back end.')
-
-    # Until mpich is default comment this out
+    variant('doc', default=False,
+            description='Build documentation.')
+    
     #depends_on('mpi@3:')
-    depends_on('mpich')
-    depends_on('mkl')
+    depends_on('mpi')
     depends_on('blaspp')
     depends_on('lapackpp')
-    depends_on('hpx cxxstd=14 networking=none')
-    depends_on('cuda', when='gpu=True')
+    depends_on('hpx@1.4.0: cxxstd=14 networking=none')
+    depends_on('cuda', when='cuda=True')
```

Comment:
Can you confirm that it is possible to check boolean variants like this?

From the CLI documentation I would say no, but maybe it is different with `depends_on`
```
variants:
      +variant                      enable <variant>
      -variant or ~variant          disable <variant>
      variant=value                 set non-boolean <variant> to <value>
      variant=value1,value2,value3  set multi-value <variant> values
```

Reply:
You're right, sorry. I forgot to try again the `+cuda` option, after moving to variant. I'm going to fix it, thanks.